CI build is failing sometime for Github rate limit

See https://ci.centos.org/job/minishift-centos-iso-pr/497/console

```
[...]
00:05:33.415 -- Checking for IP address ... OK
00:05:33.415 -- Checking if external host is reachable from the Minishift VM ... 
00:05:33.819    Pinging 8.8.8.8 ... OK
00:05:33.819 -- Checking HTTP connectivity from the VM ... 
00:05:34.207    Retrieving http://minishift.io/index.html ... OK
00:05:34.543 -- Checking if persistent storage volume is mounted ... OK
00:05:34.882 -- Checking available disk space ... 1% used OK
00:05:35.787 Error starting the cluster: Error attempting to download and cache oc: Cannot download OpenShift release asset 5165547: GET https://api.github.com/repos/openshift/origin/releases/assets/5165547: 403 API rate limit exceeded for 8.43.84.199. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.); rate reset in 10m49.649007513s
00:05:35.792 Error starting Minishift VM
00:05:35.793 make: *** [test] Error 1
00:05:35.793 Connection to 172.19.2.67 closed.
00:05:35.793 + rtn_code=2
00:05:35.794 + '[' 2 -eq 0 ']'
[...]
```

--- a/centos_ci.sh
+++ b/centos_ci.sh
@@ -10,7 +10,7 @@ set -e
 # that might interest this worker.
 if [ -e "jenkins-env" ]; then
   cat jenkins-env \
-    | grep -E "(JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId|CICO_API_KEY)=" \
+    | grep -E "(JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId|CICO_API_KEY|GITHUB_TOKEN)=" \
     | sed 's/^/export /g' \
     > ~/.jenkins-env
   source ~/.jenkins-env
@@ -53,6 +53,8 @@ PASS=$(echo $CICO_API_KEY | cut -d'-' -f1-2)
 
 rm -rf build/bin # Don't upload bin folder
 set +x
+# Export GITHUB_ACCESS_TOKEN to prevent Github rate limit
+export GITHUB_ACCESS_TOKEN=$GITHUB_TOKEN
 # For PR build, GIT_BRANCH is set to branch name other than origin/master
 if [[ "$GIT_BRANCH" = "origin/master" ]]; then
   # http://stackoverflow.com/a/22908437/1120530; Using --relative as --rsync-path not working

